tslint.json

```diff
@@ -0,0 +1,31 @@
+{
+  "rules": {
+    "class-name": true,
+    "comment-format": [true, "check-space"],
+    "indent": [true, "spaces"],
+    "no-duplicate-variable": true,
+    "no-eval": true,
+    "no-internal-module": true,
+    "no-trailing-whitespace": true,
+    "no-var-keyword": true,
+    "one-line": [true, "check-open-brace", "check-whitespace"],
```

Comment:
@davidrus look at rules here https://palantir.github.io/tslint/rules/one-line/ it means that open brace will be on the same line as preceding expression